docs/ancestry.md

```diff
@@ -393,6 +393,8 @@ Note that this is the **default**, and can be overridden by the
 {attr}`.SampleSet.ploidy` value, as described in the
 {ref}`sec_ancestry_samples_sampling_details` section.
 
+(sec_ancestry_ploidy_coalescent_time_scales)=
+
```

Comment:
suggested changes below: 
- "affects the time scale on which coalescence occurs, since it affects the total number of genomes in the population
   (see also {ref}`population size<sec_ancestry_demography>`)."
- "The resulting tree sequences are identical, except that node time differ by a factor of 2, because in the underlying coalescent algorithm the total number of genomes in the population acts as a time scaling."

Reply:
done